src/fundus/publishers/us/__init__.py

```diff
@@ -42,7 +42,14 @@ class US(PublisherEnum):
     TheIntercept = PublisherSpec(
         name="The Intercept",
         domain="https://www.theintercept.com/",
-        sources=[RSSFeed("https://theintercept.com/feed/?rss")],
+        sources=[
+            RSSFeed("https://theintercept.com/feed/?lang=en"),
+            Sitemap(
+                "https://theintercept.com/sitemap_index.xml",
+                reverse=True,
+                sitemap_filter=inverse(regex_filter("post-sitemap")),
+            ),
+        ],
```

Comment:
That this is displayed as a change here confuses me since it is already part of the master branch. There is no merge commit in this branch's commit history so I'm wondering how these changes got applied.

Reply:
I think I vaguely remember what happened there. Since I created a local branch some time ago working on this, I might have had an old version of the master branch I started off with. For some reason having it the way it was resulted in a crash (maybe it was the validators issue, I'm not too sure atm) but I cross referenced my current branch with the master branch and that seemed to be causing the problem, so I updated my local branch accordingly